src/link/object.c

```diff
@@ -305,14 +303,24 @@ static void readPatch(FILE *file, struct Patch *patch, char const *fileName, cha
 		     feof(file) ? "Unexpected end of file" : strerror(errno));
 }
 
+/**
+ * Sets a patch's pcSection from its pcSectionID.
+ * @param patch The struct to fix
+ */
+static void fixPatchSection(struct Patch *patch, struct Section *fileSections[])
```

Comment:
"Fix" implies that it was broken before, so I'd prefer a name that better conveys that it's actually filling that data in, and for the first time.

Reply:
`linkPatchToPCSect`?

Reply:
Sounds good, yep